mod-circulation/src/main/resources/vega/mod-circulation/features/requests.feature

```diff
@@ -464,3 +464,51 @@ Feature: Requests tests
     * def extRequestId = call uuid1
     * def extRequestType = 'Recall'
     * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostRequest') { requestId: #(extRequestId), itemId: #(extItemId), requesterId: #(extUserId), extRequestType: #(extRequestType), extInstanceId: #(instanceId), extHoldingsRecordId: #(holdingId) }
+
+    # This scenario does not cover testing for item with status 'Available in ASR' due to lack of implementation
+  Scenario Outline: Requests: Given an item Id, a user Id, and a pickup location, attempt to create a hold request when the applicable request policy allows holds and item status is not "Available", "Recently returned", "In process (not requestable)", "Declared lost", "Lost and paid", "Aged to lost", "Claimed returned", "Missing from ASR", "Long missing", "Retrieving from ASR", "Withdrawn", "Order closed", "Intellectual item", "Unavailable", or "Unknown"
```

Comment:
This test is not to check failing cases, please write passing cases for all not provided statuses as FAT-1038

Reply:
Sorry, did not understand you. Could you indicate line number of the test asserts failing cases?

Reply:
please check this link [here](https://folio-project.slack.com/archives/CD1CCKE4V/p1643211701164700)

Reply:
I have checked the correspondence. And the test  is about passing cases for all not provided statuses..